src/Facebook/InstantArticles/Client/Client.php

```diff
@@ -98,6 +98,17 @@ public function importArticle($article, $takeLive = false)
     }
 
     /**
+     * Removes an article from your Instant Articles library.
+     *
+     * @param string $canonicalURL The canonical URL of the article to get the status for.
+     */
+    public function removeArticle($canonicalURL) {
+        if ($articleID = $this->getArticleIDFromCanonicalURL($canonicalURL)) {
```

Comment:
Regarding `InstantArticleStatus`, I was wondering why you do a custom not found status, since PHP will already issue an error if you don't pass a param (if we had to do this for every param without defaults for every function in PHP it would get pretty cumbersome). But if that's a standard you use for some other purpose it's your SDK so I will add it. Just curious why this is done.